_data/strings.yml

```
@@ -14,6 +14,14 @@ locales:
     made_possible: "Dieses Projekt wird von den <a href='https://github.com/qmk'>QMK-Entwicklern</a> und Mitwirkenden wie Ihnen unterstützt!"
     hosted_on: "Betrieben mit GitHub Pages &mdash; Layout von <a href='https://github.com/orderedlist'>orderedlist</a>"
     description: "Eine Open Source Firmware für auf AVR oder ARM basierende Tastaturen"
+     hi: &DEFAULT_HI
```

Can you fix the indent here?

```suggestion
  hi: &DEFAULT_HI
```